flight/Modules/Sensors/sensors.c

```diff
@@ -163,12 +163,14 @@ int32_t SensorsInitialize(void)
 		case OPTICALFLOWSETTINGS_SENSORTYPE_PX4FLOW:
 #if defined(PIOS_INCLUDE_PX4FLOW)
 		{
-			struct pios_px4flow_cfg pios_px4flow_cfg;
-			pios_px4flow_cfg.rotation.roll_D100 = opticalFlowSettings.SensorRotation[OPTICALFLOWSETTINGS_SENSORROTATION_ROLL];
-			pios_px4flow_cfg.rotation.pitch_D100 = opticalFlowSettings.SensorRotation[OPTICALFLOWSETTINGS_SENSORROTATION_PITCH];
-			pios_px4flow_cfg.rotation.yaw_D100 = opticalFlowSettings.SensorRotation[OPTICALFLOWSETTINGS_SENSORROTATION_YAW];
-			if (PIOS_PX4Flow_Init(&pios_px4flow_cfg, external_i2c_adapter_id) != 0) {
-				// set alarm
+			if (external_i2c_adapter_id) {
```

Comment:
It makes sense to alarm this even without an adaptor on normal targets doesn't it? That is one of the potential causes of PX4 not working despite being selected...